src/components/compound/DlSlider/DlSlider.vue

```diff
@@ -256,6 +248,14 @@ export default defineComponent({
             onChange,
             handleResetButtonClick
         }
+    },
+    methods: {
+        handleFocus(evt: Event) {
```

Comment:
In all of the handlers parameters - Avoid of shortening variable names in this form, either shorten event to `e` or just type `event`.